workflow/controller/controller.go

```diff
@@ -78,11 +82,12 @@ type WorkflowController struct {
 	podQueue              workqueue.RateLimitingInterface
 	completedPods         chan string
 	gcPods                chan string // pods to be deleted depend on GC strategy
-	throttler             Throttler
+	throttler             sync.Throttler
 	session               sqlbuilder.Database
 	offloadNodeStatusRepo sqldb.OffloadNodeStatusRepo
 	hydrator              hydrator.Interface
 	wfArchive             sqldb.WorkflowArchive
+	concurrencyMgr        *sync.SyncManager
```

Comment:
Minor, but shouldn't it be?
```suggestion
	syncManager        *sync.SyncManager
```

Reply:
agree.